Tests/test_imagecms.py

```diff
@@ -205,7 +205,7 @@ def test_lab_color(self):
 
         target = Image.open('Tests/images/hopper.Lab.tif')
 
-        self.assert_image_similar(i, target, 30)
+        self.assert_image_similar(i, target, 3.5)
```

Comment:
New implementation much accurate for LAB colors because of signed data model.
The old implementation treat the distance between `0` and `-1` as 255